src/storage/mvcc/txn.rs

```diff
@@ -869,9 +866,9 @@ mod tests {
         // should read pending locks
         must_get_err(&engine, k1, 7);
         // should ignore the primary lock and get none when reading the latest record
-        must_get_none(&engine, k1, u64::MAX);
```

Comment:
Just for curiosity, are module level constants like this deprecated after corresponding const fns are supported?

Reply:
I don't think they are explicitly deprecated (yet), but there is no point in using them over a const fn